zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -93,6 +94,12 @@ public Factory(HttpClient httpClient) {
   }
 
   @Override protected V doExecute() throws IOException {
+    for (EventExecutor eventLoop : httpClient.factory().eventLoopGroup()) {
```

Comment:
very good idea though ISE is usually state that can resolve and I doubt this would. maybe just runtimeexception? ps I dont mind backfilling tests but I am losing track. can you add TODO testme on things like this?